doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -954,16 +975,14 @@ root@sonic:~# show interfaces status
  Ethernet24  2048,2049,2050,2051,2052,2053,2054,2055     400G   9100    N/A     etp3  routed    down       up     N/A           N/A
  Ethernet32  1792,1793,1794,1795,1796,1797,1798,1799     400G   9100    N/A     etp4  routed    down       up     N/A           N/A
  Ethernet40  1800,1801,1802,1803,1804,1805,1806,1807     400G   9100    N/A     etp5  routed    down       up     N/A           N/A
- ...
- ...
-
-### These are internal DPU ports.  Use role type to indicate NPU-DPU Data Port.###
 ...
 ...
-Ethernet192                                   4,5,6,7     200G   9100    N/A   dpu-0  routed    down       up     DPU-NPU Data Port  N/A
-Ethernet200                                   0,1,2,3     200G   9100    N/A   dpu-1  routed    down       up     DPU-NPU Data Port  N/A
-Ethernet208                                 8,9,10,11     200G   9100    N/A   dpu-2  routed    down       up     DPU-NPU Data Port  N/A
-Ethernet216                               12,13,14,15     200G   9100    N/A   dpu-3  routed    down       up     DPU-NPU Data Port  N/A
+...
```

Comment:
@rameshraghupathy "show interface status" is used to display the front panel ports. When executed on DPU host which has not front panel ports why is it showing Ethernet0 in the output?

Reply:
"show interface status" can display both front panel ports and the back panel dataplane ports also. In the case of a DPU it has one BP port and displays it.  With the introduction of the "role" in the config going forward the the Type would be "Dpc"